src/screens/Docs/Browsers.js

```diff
@@ -20,11 +20,13 @@ const children = `
 `;
 
 const Browsers = () => (
-  <Page>
-    <MarkdownTemplate name={name} desc={desc}>
-      {children}
-    </MarkdownTemplate>
-  </Page>
+  <div style={{ minHeight: '100vh' }}>
+    <Page>
```

Comment:
Can we just add the height to the page component instead of having an extra div?
Also it looks like there are a few other pages in screens/Docs that could also benefit from having the minHeight